### PR TITLE
Feature/restrictions

### PR DESCRIPTION
**Pull request by AndreasBC99**
Migrations and DbInitilazor is still a part of test coverage. Solution is coming up.

### CODE REVIEW

File: src/Chirp.DBService/Chirp.DBService.csproj [hunk 1]
```diff
@@ -13,13 +13,10 @@
...
       <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="7.0.12" />
-    </ItemGroup>
-
-
-
+    </ItemGroup>    
+    
     <ItemGroup>
-      <ProjectReference Include="..\Chirp.Utilities\Chirp.Utilities.csproj" />
+        <ExcludeFromCodeCoverage Include="src\Chirp.DBService\Migrations"/>
```
Comment (marcusaandahl) [inline src/Chirp.DBService/Chirp.DBService.csproj:19]: We also want to exclude our `Chirp.DBService\DbInitializer.cs`, as DB seeding is not something we need to test.

File: src/Chirp.DBService/Contexts/ChirpDBContext.cs [hunk 2]
```diff
@@ -21,6 +19,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             .WithMany(a => a.Cheeps)
             .HasForeignKey("AuthorId")
             .IsRequired();
+        modelBuilder.Entity<Author>().Property(a => a.Name).HasMaxLength(50).IsRequired(); //Restriction of maximum length
+        modelBuilder.Entity<Author>().Property(e => e.Email).HasMaxLength(50).IsRequired(); //Restriction of maximum length
```
Comment (marcusaandahl) [inline src/Chirp.DBService/Contexts/ChirpDBContext.cs:23]: Is it possible to research whether or not we can also add a regex restriction that the email is a valid email?

**Review by marcusaandahl — CHANGES_REQUESTED** [review]
The checks are failing because the `Chirp.Utilities` and `Chirp.Utilities.Tests` have not been properly deleted. They need to be removed.

Furthermore, now that we have changed the schema for the database, it would probably be wiser to keep our current migrations (`InitialCreate`), and then add a new migration (fx `AddAuthorRestrictions`), such that we have a clear history of our database schema migrations. This is usually best practice to do.

In short, here's what's needed to do:
- Remove Utilities projects
- Add migrations on top of our old ones
- The other comments from the review

File: src/Chirp.DBService/Contexts/ChirpDBContext.cs (hunk 2 above)
Comment (AndreasBC99) [reply src/Chirp.DBService/Contexts/ChirpDBContext.cs:23]: On it!
